agent/util-scripts/pbench-move-results

```diff
@@ -206,8 +200,28 @@ for dir in `/bin/ls -ort -d */ | awk '{print $8}' | grep -v "^tools-" | grep -v
         echo "$prefix" > $prefixfile
     fi
 
+    echo $hostname > $tarball.hostname
+
+    # pack everything in one tarball
+    tar czf complete-$tarball $tarball $tarball.md5 $prefixfile $tarball.hostname
+    
+    hash=$(md5sum "$tarball" | awk '{ print $1 }')
```

Comment:
No: the object of the game is to have `pbench-move-results` be ignorant of the details of how we store things on the server. Only the server side should know those details. This code explicitly exposes those details to the agent side. Please get rid of it.

The only thing that `pbench-move-results` should do is create the tarball and copy it to a specified directory on the server (the directory specified in the config file). That's *all* it should do.

The rest should be up to the server. That will require changes to the server of course and those changes need to be added to this PR.